common/src/main/java/com/amazonaws/services/schemaregistry/common/configs/UserAgents.java

```diff
@@ -0,0 +1,8 @@
+package com.amazonaws.services.schemaregistry.common.configs;
+
+public class UserAgents {
+    public static final String KAFKA = "kafka";
+    public static final String KAFKASTREAMS = "kafkastreams";
+    public static final String KAFKACONNECT = "kafkaconnect";
+    public static final String FLINK = "flink";
```

Comment:
Kinesis ?

Reply:
KCL and KPL will be passed from the respective libraries. I didn't want those repos to depend on this class for simple constants.

Reply:
We have anyways added a dependency there. A class unless loaded dynamically wont make a difference. Also, it gives us a streamlined option.